pages/services/data-science-engine/1.0.0/hdfs/index.md

```diff
@@ -94,48 +94,28 @@ docker run -it mesosphere/hdfs-client:1.0.0-2.6.0 bash
 ./bin/hdfs dfs -mkdir /history
 ```
 
-1. Create `spark-history-options.json`:
+1. Configure "Spark EventLog Directory" to point to the created HDFS directory in `service.json`:
 
 ```json
 {
-  "service": {
-   "hdfs-config-url": "http://api.hdfs.marathon.l4lb.thisdcos.directory/v1/endpoints"
-              }
+  "spark": {
+   "spark_history_fs_logdirectory": "hdfs://hdfs/history"
+   }
 }
 ```
-
-## Installing Spark history server
-
-1. To install the Spark history server:
-
-```bash
-dcos package install spark-history --options=spark-history-options.json
-```
-
-1. Create `spark-dispatcher-options.json`:
+1.  Enable the Spark Event log and set the HDFS directory:
 
 ```json
 {
-"service": {
-"spark-history-server-url": "http://<dcos_url>/service/spark-history"
-},
-"hdfs": {
-"config-url": "http://api.hdfs.marathon.l4lb.thisdcos.directory/v1/endpoints"
-}
+  "spark": {
+   "spark_eventlog_enabled": true,
```

Comment:
```suggestion
    "spark_eventlog_enabled": true,
```

Reply:
Your JSON is slightly off with only one space between levels. I added one extra one.

Btw, can we group all the configs together since they all fall under `spark`? (including the one [above](https://github.com/mesosphere/dcos-docs-site/pull/2456/files#diff-1a953ce258ff1442629ba630b861c0cdR102))

Reply:
Yes, sounds good! I made them separate just to show exact properties that need to be configured for each step. I updated the second code snippet to include complete required configuration.